contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/gobblin/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.github.johnrengelman.shadow'
+
+repositories {
+    jcenter()
+    maven {
+        // For: "org.pentaho:pentaho-aggdesigner-algorithm:5.1.3-jhyde"
+        // required by: "gobblin-runtime:0.10.0 > org.apache.hive:hive-exec:1.0.1"
+        url "http://repo.spring.io/plugins-release"
```

Comment:
I'm not sure why Gobblin wants this dependency but so far I have no problem running distcp without it. We could include this dependency for completeness, but that would add a few extra pounds to the fatjar. It's already 91 MB without hive. If necessary, I can remove the `exclude`s in root build.gradle instead. Then it will reach about 100 MB. What do you think?